### PR TITLE
Use Sylius v1.1.0-RC

### PR DESCRIPTION
**Pull request by Zales0123**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "require": {
         "php": "^7.1",
 
-        "sylius/sylius": "^1.0",
+        "sylius/sylius": "^1.1",
```
Comment (lchrusciel) [inline composer.json]: Is there any reason to restrict version below 1.1?

Comment (Zales0123) [reply composer.json]: Well, no 😄

**Comment by lchrusciel** [comment]
Thanks Mateusz Zalewski! 🐋

**Comment by pamil** [comment]
Mateusz Zalewski is great, although we could run tests for both Sylius 1.0 and 1.1 (same as we do with Symfony in Sylius).

**Comment by Zales0123** [comment]
That's true Kamil Kokot! It should be definitely done for this repository. Nice suggestion!

**Comment by pamil** [comment]
All right, I've implemented it in #254. Mateusz Zalewski and Łukasz Chruściel, you might want to check that out!

**Pull request merged**